client/src/components/navigation/Navigation.jsx

```diff
@@ -1,5 +1,17 @@
-export default function Navigation(){
-    return (
-        <nav>Navigation</nav>
-    )
-}
\ No newline at end of file
+export default function Navigation() {
+  return (
+    <>
```

Comment:
Looks like React fragment is redundant here, as it only has one child `<nav>`, maybe remove?😊